src/core/linter-rules/check-punctuation.js

```diff
@@ -10,7 +10,7 @@ const punctuationMarks = [".", ":", "!", "?"];
 const humanMarks = punctuationMarks.map(mark => `"${mark}"`).join(", ");
 const meta = {
   en: {
-    description: "`<p>` tags should end with a punctuation mark.",
+    description: "\`<p>\` tags should end with a punctuation mark.",
```

Comment:
this should fix it.

Reply:
Nope :cry:

Reply:
How odd... what's the markup that's being generated now?

Reply:
```
const meta = {
    en: {
      description: "\`<p>\` tags should end with a punctuation mark.",
      howToFix: `Please make sure \`<p>\` tags end with one of: ${humanMarks}.`,
    },
};
```
This is still rendering :- 
![screenshot from 2018-04-10 07-24-26](https://user-images.githubusercontent.com/25258877/38531962-3df3acf2-3c90-11e8-9286-c5ba57174a63.png)